src/PhpSpec/Console/Command/DescribeCommand.php

```diff
@@ -64,9 +67,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $container = $this->getApplication()->getContainer();
         $container->configure();
 
-        $classname = $input->getArgument('class');
-        $resource  = $container->get('locator.resource_manager')->createResource($classname);
+        if ($input->getArgument('class')) {
+            $classname = $input->getArgument('class');
+        } else {
+            $questionHelper = $this->getApplication()->getHelperSet()->get('question');
+            $question = new Question('<info>Enter class to describe: </info>');
```

Comment:
To be honest I was thinking about something more simple like adding the following method to `\PhpSpec\Console\Prompter` interface:
```php
public function askQuestion(string $question, array $autocompleteValues = []): string;
```

and implementing it in `\PhpSpec\Console\Prompter\Question` (it already has access to symfony `QuestionHelper`).
But creating hierarchy of questions may be useful too.

I don't want to force you to specific design choice, just wanted to make sure that new console-specific code will be  reusable and testable.